fiftyone/utils/voc.py

```diff
@@ -580,8 +582,18 @@ def to_detection(self, frame_size, extra_attrs=True):
         label = self.name
         bounding_box = self.bndbox.to_detection_format(frame_size)
 
+        # Handles CVAT exported attributes
+        cvat_attrs = self.attributes.get("attributes", None)
```

Comment:
I think it makes more sense to do this attribute parsing in `__init__()` so that if we add other `to_*()` methods they don't have to re-implement this logic, which they would presumably also require